[2025-05-20T18:56:55.348+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-20T18:56:57.256+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-05-20T18:56:52.092396+00:00 [queued]>
[2025-05-20T18:56:57.265+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-05-20T18:56:52.092396+00:00 [queued]>
[2025-05-20T18:56:57.265+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-20T18:56:57.279+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): TestConnectionDB> on 2025-05-20 18:56:52.092396+00:00
[2025-05-20T18:56:57.296+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2719) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-20T18:56:57.298+0000] {standard_task_runner.py:72} INFO - Started process 2720 to run task
[2025-05-20T18:56:57.299+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'BinanceCryptoAPI', 'TestConnectionDB', 'manual__2025-05-20T18:56:52.092396+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/ETL/BinanceETL.py', '--cfg-path', '/tmp/tmpd30qp29_']
[2025-05-20T18:56:57.303+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask TestConnectionDB
[2025-05-20T18:56:57.408+0000] {task_command.py:467} INFO - Running <TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-05-20T18:56:52.092396+00:00 [running]> on host 5255e1c5bd06
[2025-05-20T18:56:57.594+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BinanceCryptoAPI' AIRFLOW_CTX_TASK_ID='TestConnectionDB' AIRFLOW_CTX_EXECUTION_DATE='2025-05-20T18:56:52.092396+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-20T18:56:52.092396+00:00'
[2025-05-20T18:56:57.598+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-20T18:56:57.633+0000] {base.py:84} INFO - Retrieving connection 'DBPostgres'
[2025-05-20T18:56:57.641+0000] {logging_mixin.py:190} INFO - Connection to database successful
[2025-05-20T18:56:57.643+0000] {sql.py:511} INFO - Running statement: SELECT * FROM cryptoInfo, parameters: None
[2025-05-20T18:56:57.664+0000] {sql.py:520} INFO - Rows affected: 3106
[2025-05-20T18:56:57.664+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-20T18:56:57.673+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-20T18:56:57.674+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=BinanceCryptoAPI, task_id=TestConnectionDB, run_id=manual__2025-05-20T18:56:52.092396+00:00, execution_date=20250520T185652, start_date=20250520T185657, end_date=20250520T185657
[2025-05-20T18:56:57.750+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-20T18:56:57.796+0000] {taskinstance.py:3901} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-05-20T18:56:57.801+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
